spacy/lookups.py

```diff
@@ -118,6 +117,21 @@ def to_disk(self, path, **kwargs):
             with filepath.open("wb") as file_:
                 file_.write(self.to_bytes())
 
+    def from_disk(self, path, **kwargs):
+        """Load lookups from a directory containing a lookups.bin.
+
+        path (unicode / Path): The file path.
+        RETURNS (Lookups): The loaded lookups.
+        """
+        path = ensure_path(path)
+        filepath = path / "lookups.bin"
+        if filepath.exists():
+            data = srsly.read_msgpack(filepath)
+            for key, value in data.items():
+                self._tables[key] = Table(key)
+                self._tables[key].update_raw(value)
+        return self
+
     def from_disk(self, path, **kwargs):
```

Comment:
Should this `from_disk` method go? There are two methods here, so it looks like the method added here isn't actually used?